index.css

```diff
@@ -0,0 +1,18 @@
+/* Error: expected "{".
+ *    ,
+ * 21 |   &.active;
```

Comment:
This line indicates a syntax error in the SCSS file. The `&.active;` line is incorrect because it uses a semicolon instead of a block. In SCSS, the `&` is used to reference the parent selector, and it should be followed by a block of styles enclosed in curly braces `{}`. You should check the SCSS file and correct this by replacing the semicolon with a block of styles.